src/sentry/incidents/action_handlers.py

```diff
@@ -133,15 +133,16 @@ def _get_targets(self) -> Set[int]:
         elif self.action.target_type == AlertRuleTriggerAction.TargetType.TEAM.value:
             users = None
             if should_use_notifications_v2(self.project.organization):
-                controller = NotificationController(
-                    recipients={RpcUser(id=member.user_id) for member in target.member_set},
-                    project_ids=[self.project.id],
+                out = get_notification_recipients_v2(
+                    recipients=RpcActor.many_from_object(
```

Comment:
@GabeVillalobos def could use eyes on this line, is this the right way?

Reply:
Hmmm what is the type for `target.member_set`? It may be better to just add a new implementation within the `many_from_object` method to handle this type rather than constructing temporary RpcUsers with just IDs 🤔

Reply:
@GabeVillalobos I realize I can just make `RpcActor` directly, just pushed a change